playwright/test/search-works.test.ts

```diff
@@ -1,336 +1,282 @@
-    // the first page.
-    await newWorksSearch(context, page);
-    await searchFor('Persian', page);
-    await openDropdown('Formats', page);
-    await selectCheckbox('Archives and manuscripts', page);
-    await navigateToNextPage(page);
-
-    // Save the URL of the current search page, which will be something like
-    // https://www-stage.wellcomecollection.org/search/works?query=Persian&workType=h&page=2
-    const originalSearchUrl = new URL(page.url());
-
-    // Now go to the third result on the page, and look for "Back to search results".
-    await navigateToResult(3, page);
-
-    const backToSearchResults = await page.$(
-      'a:has-text("Back to search results")'
-    );
-    expect(backToSearchResults).toBeTruthy();
-
-    // Note: we expect the URLs on the "Back to search results" link to be relative,
-    // but the URL class only takes absolute URLs.
-    //
-    // Check this is a relative URL, then construct an absolute URL we can compare.
-    const backToSearchResultsUrlString =
-      (await backToSearchResults?.getAttribute('href')) as string;
-    expect(backToSearchResultsUrlString.startsWith('/')).toBe(true);
-
-    const backToSearchResultsUrl = new URL(
-      `${process.env.PLAYWRIGHT_BASE_URL}${backToSearchResultsUrlString}`
-    );
-
-    // Now compare the URLs.  Note that the query parameters may be in a different order,
-    // but they're still equivalent for our purposes, e.g.
-    //
-    //      /search/works?query=Persian&workType=h&page=2 and
-    //      /search/works?query=Persian&page=2&workType=h
-    //
-    // are both totally fine.  Sorting them first makes them easier to compare.
-    expect(originalSearchUrl.pathname).toEqual(backToSearchResultsUrl.pathname);
-
-    originalSearchUrl.searchParams.sort();
-    backToSearchResultsUrl.searchParams.sort();
-
-    expect(originalSearchUrl.searchParams).toEqual(
-      backToSearchResultsUrl.searchParams
-    );
-  });
+  expect(originalSearchUrl.searchParams).toEqual(
+    backToSearchResultsUrl.searchParams
+  );
 });
 
-test.describe('Scenario 2: The person is searching for a work on open shelves', () => {
-  test('the work should be browsable to from the search results', async ({
-    page,
-    context,
-  }) => {
-    await newWorksSearch(context, page);
-    await searchFor('eyes', page);
-    await openDropdown('Locations', page);
-    await selectCheckbox('Open shelves', page);
-    await navigateToNextPage(page);
-
-    expectSearchParam('availabilities', 'open-shelves', page);
-
-    await navigateToResult(6, page);
-  });
+test('(2) | The user is searching for a work on open shelves; it should be browsable from the search results', async ({
+  page,
+  context,
+}) => {
+  await newWorksSearch(context, page);
+  await searchFor('eyes', page);
+  await selectFilter('availabilities', 'open-shelves', page); // Locations > Open shelves
+  await navigateToNextPage(page);
+
+  await expect(
```

Comment:
This tests in this file are a bit confusing - I am not convinced that the assertions we're using are the best representation of what we actually want to test. In fact, I'm not sure exactly what we want to test here...

Currently we say:
1. We can make a search
2. We can select a filter for open shelves
3. We can go to the next page
4. We still have the filter in the URL
5. We can click on the 6th result on this 2nd page

Which seems quite arbitrary and quite dependent on the contents of the API... What do you think the intention of these tests is, and what assertions do you think best test that intention?

Reply:
I wonder if there were issues with open shelves results as _some_ point? These tests will have been taken from the former `works.test` file I believe so it might be OK to remove if we don't feel like it's useful. Same for scenario 3-4-5. 
1.1 and 12 would cover the "you can search and navigate to" scenarios.

Reply:
Also this comment I just reread for the bit where it navigates to a random work:
> // This is a check that we have actually loaded some results from
  // the API, and the API hasn't just errored out.

Reply:
In that case we should should test for that, rather than whether we can click on the 6th result: eg we could test that there are more than zero results available.